benchmark/profile_restful_api.py

```diff
@@ -138,10 +152,21 @@ def process_request(self,
 
         start = time.time()
 
+        if img_hw is not None:
+            import PIL
+
+            from lmdeploy.vl.utils import encode_image_base64
+            h, w = [int(s) for s in img_hw.split('*')]
```

Comment:
can we use "x" instead of "*"? "x" just press one key, "*" needs to press shift+8